modules/catalog/src/main/java/org/apache/ignite/internal/catalog/descriptors/CatalogDescriptorUtils.java

```diff
@@ -188,4 +211,21 @@ private static CatalogIndexColumnDescriptor toIndexColumnDescriptor(IndexColumnV
 
         return new CatalogIndexColumnDescriptor(config.name(), collation);
     }
+
+    // TODO: IGNITE-19719 Fix it
+    private static CatalogDataStorageDescriptor toDataStorageDescriptor(DataStorageView config) {
+        String dataRegion;
+
+        try {
+            Method dataRegionMethod = config.getClass().getMethod("dataRegion");
```

Comment:
Yes, this "great" solution came from the unavailability of these engines, and at the moment I would not really like to add them to the dependency, since it will turn out to be a cyclic dependency.

I created IGNITE-19719 to figure out how we can do well.